<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorSettingsPage</class>
 <widget class="QWidget" name="EditorSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>9</number>
      </property>
      <item row="0" column="1" colspan="2">
       <widget class="QFontComboBox" name="fontCmb"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="fontSizeSpin">
        <property name="minimum">
         <number>6</number>
        </property>
        <property name="maximum">
         <number>24</number>
        </property>
        <property name="value">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Indentation Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>9</number>
      </property>
      <item>
       <widget class="QCheckBox" name="autoDetectIndentationChk">
        <property name="toolTip">
         <string>When loading a document, try to automatically 
detect and set its indentation settings</string>
        </property>
        <property name="text">
         <string>Auto-detect document indentation</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="unindentChk">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>If a document uses spaces for indentation, 
the current option lets backspace key to remove the 
whole indent instead of removind single whitespace</string>
        </property>
        <property name="text">
         <string>Backspace removes indent</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="showLineLengthChk">
          <property name="text">
           <string>Line length indicator</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="lineLengthSpin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> symbols</string>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>60</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Document Indentation Defaults</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>-1</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Tabstop width</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="tabStopWidthSpin">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="replaceTabsChk">
        <property name="toolTip">
         <string>Use spaces for indentation instead of tabs</string>
        </property>
        <property name="text">
         <string>Replace tabs with spaces</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>showLineLengthChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineLengthSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
